scripts/test.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
```

Comment:
instead of test.sh, it could run_test.sh or something. test.sh is very common name while doing some scratch work.